### PR TITLE
Feat/mc v1 upgrade

### PR DESCRIPTION
**Pull request by cjpillsbury**
Good to go. If we need to do a hot fix release before mc@1.0 is out of rc, we're comfortable releasing against the latest rc.

### CODE REVIEW

File: packages/mux-player/src/template.ts [hunk 1]
```diff
@@ -16,12 +16,14 @@ const muxTemplate = document.createElement('template');
 if ('innerHTML' in muxTemplate) muxTemplate.innerHTML = muxTheme;
 
 // prettier-ignore
-export const template = (props: MuxTemplateProps) => html`
+export const template = (props: MuxTemplateProps) => {
+  console.log('props!', props);
```
Comment (luwes) [inline packages/mux-player/src/template.ts:20]: don't forget to remove

File: packages/mux-player/src/media-theme-mux.html [hunk 2]
```diff
@@ -53,8 +53,8 @@
       width: 100%;
     }
 
-    [disabled],
-    [aria-disabled='true'] {
+    [disabled]:not(media-live-button),
+    [aria-disabled='true']:not(media-live-button) {
```
Comment (cjpillsbury) [inline packages/mux-player/src/media-theme-mux.html:57]: NOTE: Unrelated fix for styling of live button (which is disabled when at the live edge)

File: packages/mux-player/src/index.ts [hunk 3]
```diff
@@ -69,24 +69,24 @@ const PlayerAttributes = {
 
 const ThemeAttributeNames = [
   'audio',
-  'backward-seek-offset',
-  'default-show-remaining-time',
-  'default-showing-captions',
+  'backwardseekoffset',
+  'defaultshowremainingtime',
+  'defaultshowingcaptions',
```
Comment (luwes) [inline packages/mux-player/src/index.ts:74]:
```suggestion
  'defaultshowingsubtitles',
```
